generator/operation.go

```diff
@@ -271,7 +271,7 @@ type codeGenOpBuilder struct {
 	DefaultScheme       string
 	DefaultProduces     string
 	DefaultConsumes     string
-	Security            []analysis.SecurityRequirement
+	Security            [][]analysis.SecurityRequirement
```

Comment:
big contribution toward #1214 , but not solved yet: since the original spec structure is a map, the inner [] is still random and need sorting here.

Reply:
requirements aren't a map, the order is stable there. The only time a map structure is used is when describing a requirement is as a set for deduping but the items should be appended in a consistent order.

Reply:
I was referring to the OperationProps.Security in spec (https://github.com/go-openapi/spec/blob/bcff419492eeeb01f76e77d2ebc714dc97b607f5/operation.go#L34). This is an array of maps.

Reply:
Same in https://github.com/go-openapi/spec/blob/bcff419492eeeb01f76e77d2ebc714dc97b607f5/swagger.go#L84

Reply:
that means the requirements with "AND" (keys) come in any order right?